src/lib/form-field/form-field.ts

```diff
@@ -94,7 +94,7 @@ export class MdFormField implements AfterViewInit, AfterContentInit, AfterConten
 
   /** @deprecated Use `color` instead. */
   @Input()
-  get dividerColor() { return this.color; }
+  get dividerColor(): 'primary' | 'accent' | 'warn' { return this.color; }
```

Comment:
We have the `ThemePalette` type for this already.

Reply:
Yeah I wanted to use that as well, but it also includes `undefined` as a possible value, which is not valid here.